recipe/run_test.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+"""
+Test script for openjpeg.
+
+Currently main purpose is to check that the openjpeg library
+if linked dynamically to libpng, libtiff and libz as in 
+the latest build recipe.
+"""
+
+import subprocess
+
+# command we run to get output - for subprocess.check_output
+INSPECT_COMMAND = ['conda', 'inspect', 'linkages', 'openjpeg', '--name', '_test']
+
+# Currently we only check that this string exists in the output
+# of INSPECT_COMMAND.
+# This allows some difference in the endings (versions, extension etc)
+# but may need to be tightened up at some stage.
+# Need to be bytes to keep Python3 happy (subprocess.check_output returns bytes)
+REQ_LIBS = (b'lib/libtiff', b'lib/libpng', b'lib/libz')
```

Comment:
That's fine. Then we just verify `$PREFIX` shows up for each on Linux. However, that will only work on Linux as Mac will probably use `@rpath` or possibly (rarely) `@loader_path`.

```
# ldd /opt/conda2/lib/libtiff.so
	linux-vdso.so.1 =>  (0x00007fff93b9c000)
	liblzma.so.5 => /opt/conda2/lib/./liblzma.so.5 (0x00007f77a3732000)
	libjpeg.so.8 => /opt/conda2/lib/./libjpeg.so.8 (0x00007f77a34fa000)
	libz.so.1 => /opt/conda2/lib/./libz.so.1 (0x00007f77a32e4000)
	libm.so.6 => /lib64/libm.so.6 (0x00007f77a305b000)
	libpthread.so.0 => /lib64/libpthread.so.0 (0x00007f77a2e3d000)
	libc.so.6 => /lib64/libc.so.6 (0x00007f77a2aa9000)
	/lib64/ld-linux-x86-64.so.2 (0x000056432e82c000)
```